internal/service/resources/serve.go

```diff
@@ -140,7 +140,7 @@ func Serve(config *api.ResourceServerConfig) error {
 		Repo:   repository,
 		Info: generated.OCloudInfo{
 			Description:   "OpenShift O-Cloud Manager",
-			GlobalCloudId: globalCloudID,
+			GlobalcloudId: globalCloudID,
```

Comment:
Probably a dumb question, but why is the leading "G" capitalized here?

Reply:
Because it is a public/exported field on the Go struct.  If you look in the `inventory.generated.go` file  you'll see the `json:"globalcloudId"` annotation which says how it will be serialized for output.

Reply:
Cool - thanks Allain